### ISSUE TITLE
Configuration error at 'projectDir': should match pattern "/^(\.\/?|(\.\/)?\.\.(\/\.\.)*\/?)$/"

### ISSUE THREAD

**Issue by PierrickI3** [serverless/serverless#9573]
Just upgraded to the latest serverless version to try the new `projectDir` but I am getting an error referencing a file outside of the current directory:

serverless.yml file (omitted some sections for clarity purposes):

```
app: saastemplate
service: system
configValidationMode: error
useDotenv: true
projectDir: ../

provider:
  name: ${file(../config.yml):name}
  runtime: nodejs14.x
...
```

Error thrown when running `serverless offline start`:

```
 Serverless Error ----------------------------------------

  Configuration error at 'projectDir': should match pattern "/^(\.\/?|(\.\/)?\.\.(\/\.\.)*\/?)$/"

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com

  Your Environment Information ---------------------------
     Operating System:          win32
     Node Version:              14.16.0
     Framework Version:         2.45.0
     Plugin Version:            5.2.0
     SDK Version:               4.2.3
     Components Version:        3.11.0
```

The projectDir `../` matches the pattern `"/^(\.\/?|(\.\/)?\.\.(\/\.\.)*\/?)$/"` though?

```
Framework Core: 2.45.0
Plugin: 5.2.0
SDK: 4.2.3
Components: 3.11.0
```

**Comment by medikoo** [comment]
@PierrickI3 great thanks for reporting. It'll be fixed by #9574 and published shortly after

**Comment by PierrickI3** [comment]
Thanks!
